mw-webapp/src/utils/FormatterUtils.ts

```diff
@@ -0,0 +1,15 @@
+const FIRST_CHAR = 1;
+
+/**
+ * Formatter utils
+ */
+export class FormatterUtils {
+
+  /**
+   * Checking for zero and formatting the value
+   */
+  public static defaultFormatter(value: string) {
+    return value.startsWith("0") ? value.slice(FIRST_CHAR) : Number(value);
+  }
```

Comment:
Do we really need a special util , maybe better to use line 12 in the EditableText on line 76? Let's discuss how will be better)